backend/controllers/prisonerProfile/prisonerProfessionalContacts.ts

```diff
@@ -97,12 +106,16 @@ export default ({ prisonApi, personService, allocationManagerApi }) =>
       })
     }
 
+    const redirectEnabled = getRedirectEnabled(res)
+    const redirectCaseload = getRedirectCaseLoad(req?.session?.userDetails.activeCaseLoadId)
+
     return res.render('prisonerProfile/prisonerProfessionalContacts/prisonerProfessionalContacts.njk', {
       breadcrumbPrisonerName: putLastNameFirst(firstName, lastName),
       contactsGroupedByRelationship: contactsGroupedByRelationship.sort((left, right) =>
         left.relationship.localeCompare(right.relationship)
       ),
       offenderNo,
       prisonerName: formatName(firstName, lastName),
+      redirectIsActive: !!(redirectEnabled && redirectCaseload),
     })
   }
```

Comment:
Is this going to happen in a lot of places? I wonder if a middleware solution would be beneficial so it doesn't litter all the controller functions